include/thingset/can.h

```diff
@@ -150,6 +188,18 @@ extern "C" {
  */
 typedef void (*thingset_can_report_rx_callback_t)(uint16_t data_id, const uint8_t *value,
```

Comment:
Wondering what the value of `data_id` would be in this callback.

Reply:
It was an idea to maintain compatibility with the callback used for items, but you are right, it does not really make sense for entire reports.

I have updated the report callback such that it provides the entire report. This can now also be used for text mode.